apps/api-gateway/src/verification/verification.controller.ts

```diff
@@ -193,19 +194,11 @@ export class VerificationController {
         if (requestType === ProofRequestType.PRESENTATIONEXCHANGE && !requestProof.presentationDefinition) {
                 throw new BadRequestException(`type: ${requestType} requires presentationDefinition`);
         }
-        if (requestProof.proofFormats) {
-            const attributeArray = [];
-        for (const attrData of requestProof.proofFormats.indy.attributes) {
-          if (0 === attributeArray.length) {
-            attributeArray.push(Object.values(attrData)[0]);
-          } else if (!attributeArray.includes(Object.values(attrData)[0])) {
-            attributeArray.push(Object.values(attrData)[0]);
-          } else {
-            throw new BadRequestException('Please provide unique attribute names');
-          }           
+        
+          if (requestProof.proofFormats) {
```

Comment:
```suggestion
          if (ProofRequestType.INDY) {
```

Both are right, however checking against type makes it more obvious.

Reply:
Yes, checking against type makes it more obvious.